tests/reward_score/test_sandbox_fusion.py

```diff
@@ -0,0 +1,540 @@
+
+    correct_count = 0
+    wrong_count = 0
+    timeout_count = 0
+    unexpected_results = []
+    for i, r in enumerate(results):
+        expected = expected_results_map[i]
+        if r == expected:
+            if expected is True:
+                correct_count += 1
+            elif expected is False:
+                wrong_count += 1
+            elif expected == -3:
+                timeout_count += 1
+        else:
+            unexpected_results.append((i, r, f"Expected {expected}"))
+
+    print(f"Correct results (True): {correct_count}/{concurrency_level - len(wrong_answer_indices) - len(timeout_indices)}")
+    print(f"Expected wrong answers (False, correctly identified): {wrong_count}/{len(wrong_answer_indices)}")
+    print(f"Expected timeouts (-3, correctly identified): {timeout_count}/{len(timeout_indices)}")
+
+    if unexpected_results:
+        print("Unexpected results found:")
+        for idx, res, expected_str in unexpected_results[:10]:  # Print first 10 unexpected
+            print(f"  Index {idx}: Got {res}, {expected_str}. Metadata: {metadata_list[idx]}")
+        raise AssertionError(f"Found {len(unexpected_results)} unexpected results.")
+
+    assert correct_count == concurrency_level - len(wrong_answer_indices) - len(timeout_indices), "Incorrect number of successful results"
+    assert wrong_count == len(wrong_answer_indices), "Incorrect number of identified wrong answers"
+    assert timeout_count == len(timeout_indices), "Incorrect number of identified timeouts"
+
+    # Verify metadata count and basic status of one of each type
+    assert len(metadata_list) == concurrency_level
+    # Find the first correct index
+    first_correct_index = next(i for i in range(concurrency_level) if i not in wrong_answer_indices and i not in timeout_indices)
+    assert metadata_list[first_correct_index]["status"] == "success"
+    assert metadata_list[first_correct_index]["stdout"] == f"output_{first_correct_index}\n"
+
+    # Check the status of the first intentionally wrong case
+    first_wrong_index = min(wrong_answer_indices)
+    assert metadata_list[first_wrong_index]["status"] == "wrong_answer"
+    assert metadata_list[first_wrong_index]["stdout"] == f"output_{first_wrong_index}\n"
+    assert metadata_list[first_wrong_index]["expected_output"] == f"wrong_output_{first_wrong_index}\n"
+
+    # Check the status of the first intentionally timeout case
+    first_timeout_index = min(timeout_indices)
+    assert metadata_list[first_timeout_index]["status"] == "timeout"
+    # For timeout, stdout might be None or empty depending on when the timeout occurred
+    # assert metadata_list[first_timeout_index]["stdout"] is None or metadata_list[first_timeout_index]["stdout"] == ""
+
+
+# --- Unit test cases (using mock) ---
+
+
+@patch("verl.utils.reward_score.sandbox_fusion.utils.call_sandbox_api")
+def test_unit_concurrency_order(mock_call_sandbox_api):
+    """单元测试：验证并发执行时结果顺序是否正确"""
```

Comment:
please leave docs in English, thank you. Same for L238, L244, L255, L257, L265, and a few more occurrences

Reply:
> please leave docs in English, thank you. Same for L238, L244, L255, L257, L265, and a few more occurrences

fixed